templates/web/base/report/new/category.html

```diff
@@ -5,7 +5,7 @@
     <label for='form_category' id="form_category_label">
         [%~ loc('Category') ~%]
     </label>[% =%]
-    <select class="form-control" name='category' id='form_category'>
+    <select class="form-control" name='category' id='form_category' data-role='[% c.user_exists AND c.user.has_body_permission_to('planned_reports') ? 'inspector' : 'user'  %]' data-body='[% c.user.from_body.name %]'>
```

Comment:
Are you sure? That seems very odd if so.
Having it there would (if it was the only use) prevent a needless lookup if not logged in, and can prevent an error in actual code, but yes, it's probably not needed in the template here.